ci/scripts/create-and-update-db.sh

```diff
@@ -21,4 +21,9 @@ for db in ${DATABASES}; do
     psql_adm -d "${db}" -c "CREATE EXTENSION IF NOT EXISTS \"${ext}\""
   done
 
+  # Special case for uaadb, create totp seed table for use with MFA
+  if [ "${db}" = "uaadb" ]; then
+    psql_adm -d "${db}" -c "CREATE TABLE IF NOT EXISTS totp_seed ( username varchar(255) PRIMARY KEY, seed varchar(36), backup_code varchar(36) )"
```

Comment:
Curiosity. Are we only going only going to be storing one backup code? My familiarity with back-up codes is that they'd be a list. Similar to Github's and Google's backup code. Either way though, this is good for now. 👍